assets/charts/control-plane/packet-ccm/templates/deployment.yaml

```diff
@@ -0,0 +1,186 @@
+---
+apiVersion: apps/v1
+kind: Deployment
+metadata:
+  name: packet-cloud-controller-manager
+  labels:
+    app: packet-cloud-controller-manager
+spec:
+  replicas: 1
+  selector:
+    matchLabels:
+      app: packet-cloud-controller-manager
+  template:
+    metadata:
+      labels:
+        app: packet-cloud-controller-manager
+      annotations:
+        scheduler.alpha.kubernetes.io/critical-pod: ''
+    spec:
+      dnsPolicy: Default
+      hostNetwork: true
+      serviceAccountName: cloud-controller-manager
+      tolerations:
+        # This taint is set by all kubelets running `--cloud-provider=external`
+        # so we should tolerate it to schedule the Packet CCM.
+        - key: "node.cloudprovider.kubernetes.io/uninitialized"
+          value: "true"
+          effect: "NoSchedule"
+        - key: "CriticalAddonsOnly"
+          operator: "Exists"
+        # Cloud controller manager should be able to run on masters.
+        - key: "node-role.kubernetes.io/master"
+          effect: NoSchedule
+      containers:
+      # Use our own image until https://github.com/packethost/packet-ccm/pull/101 and
+      # https://github.com/packethost/packet-ccm/pull/107 are merged and released.
+      # This image must also support amd64 and arm64 architectures.
+      - image: quay.io/kinvolk/packet-ccm:kinvolk
```

Comment:
Is this built on top of 2.0.0?

Reply:
On top of latest master. Perhaps should push this tag to our fork